pages/common/dvtm.md

```diff
@@ -0,0 +1,32 @@
+# dvtm
+
+> Dynamic virtual terminal manager.
+
+- Show manual page:
+
+`Ctrl+g ?`
+
+- Create a new shell window:
+
+`Ctrl+g c`
+
+- Focus the n-th window:
+
+`Ctrl+g n`
+
+- Close focused window:
+
+`Ctrl+g x x`
```

Comment:
We do not use multiple consecutive examples. Please separate them.

Reply:
I noticed that most people don't know that ctrl+d sends \0 which the shell processes as logout. And in most cases, such a window closing is much better than killing it with ctrl+g x x. 
How to write this best?

Reply:
Go with the most common scenario. I think that should be Ctrl+d.

Reply:
Is it ok that Ctrl-d is not a dvtm hotkey, but little known(i think) standard hotkey for a shell?

Reply:
As I said before, choose the command you would recommend normally. This is a tldr page, so we cannot add everything. If ctrl-d is a better choice than ctrl-g x x, then go for it. Even if it's not a dvtm hotkey, if it is what the users would normally use, we should use that.

We always prefer practical and useful commands rather than sticking with a man-page like approach.